src/main/java/org/fogbowcloud/arrebol/api/http/controllers/JobController.java

```diff
@@ -25,15 +25,23 @@ public JobController(JobService jobService){
 
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<String> addJob(@RequestBody JobSpec jobSpec){
-        LOGGER.info("Arrebol: saving new Job: " + jobSpec.getLabel() + ".");
+        LOGGER.info("Arrebol: Adding new Job: " + jobSpec.getLabel() + ".");
 
-        String idJob = jobService.addJob(jobSpec);
-        JobResponse jobResponse = new JobResponse(idJob);
+        String jobId = jobService.addJob(jobSpec);
+        JobResponse jobResponse = new JobResponse(jobId);
 
-        LOGGER.info("Saved " + jobSpec.getLabel() + " with id " + idJob + ".");
+        LOGGER.info("Added " + jobSpec.getLabel() + " with id " + jobId + ".");
         return new ResponseEntity(jobResponse, HttpStatus.CREATED);
     }
 
+    @RequestMapping(value = ApiDocumentation.ApiEndpoints.JOB_PATH, method = RequestMethod.GET)
+    public ResponseEntity<String> getJob(@PathVariable String id){
+        LOGGER.info("Arrebol: Getting an job with id: " + id);
```

Comment:
@wesleymonte our log lines are not consistent. For example, in some places we Use "Arrebol:" in other places, we do not use. It is glorious to be consistent. I suggest to NOT use "Arrebol:" (obviously, it is arrebol log file)

NOTE: we do not need to change this comment in this PR, however, it would be nice to change it in the next commits